packages/keyutils/keyutils.spec

```
@@ -0,0 +1,85 @@
+Name: %{_cross_os}keyutils
+Version: 1.6.1
+Release: 1%{?dist}
+Summary: Linux Key Management Utilities
+License: GPLv2+ and LGPLv2+
```

We use SPDX identifiers and expressions in the license field, so this should be:
```suggestion
License: GPL-2.0-or-later AND LGPL-2.1-or-later
```